operations/mimir/autoscaling.libsonnet

```diff
@@ -215,10 +257,20 @@
     if !$._config.autoscaling_querier_enabled then {} else removeReplicasFromSpec
   ),
 
+  query_frontend_scaled_object: if !$._config.autoscaling_query_frontend_enabled then null else
```

Comment:
For testing purposes, please enable both query-frontend and ruler-query-frontend autoscaling in `operations/mimir-tests/test-autoscaling.jsonnet`.

Reply:
Thanks for the tip, wasn't aware.

Reply:
Done, PTAL,.